.github/workflows/db_production_flask_migration_manual.yaml

```diff
@@ -1,7 +1,19 @@
-name: Production - Manual DB Downgrade
+name: Production - Manual DB Flask Command
 
 on:
   workflow_dispatch:
+    inputs:
+      password:
+        description: "Enter the password - can be found here: https://start.1password.com/open/i?a=BZ4RYZZICJEKNOF4Z6ASRX2XXI&v=ppnxsriom3alsxj4ogikyjxlzi&i=i5kfypu7grhf5lc2bu4mi4yque&h=cds-snc.1password.ca"
```

Comment:
Can we put in a link in here? markdown format?

Reply:
I had the same thought -- quickly couldn't find a way, but I'll look deeper